main.go

```diff
@@ -35,42 +36,178 @@ import (
 	"sync"
 	"time"
 
+	"github.com/fatih/color"
+	ui "github.com/gizak/termui/v3"
+	"github.com/gizak/termui/v3/widgets"
+	"github.com/gorilla/mux"
 	"github.com/minio/cli"
 	"github.com/minio/minio/pkg/console"
 	"github.com/minio/minio/pkg/ellipses"
 )
 
 const slashSeparator = "/"
 
+var (
+	globalQuietEnabled   bool
+	globalDebugEnabled   bool
+	globalLoggingEnabled bool
+	globalTraceEnabled   bool
+	globalJSONEnabled    bool
+	globalConsoleDisplay bool
+	globalTermTable      *widgets.Table
+	globalConnStats      []*ConnStats
+	timeZero             = time.Time{}
+)
+
+// Standard env prometheus auth type
+const (
+	EnvPrometheusAuthType = "SIDEKICK_PROMETHEUS_AUTH_TYPE"
```

Comment:
do we need auth type env? I am assuming this will always be public

Reply:
removed this.